rotkehlchen/externalapis/helius.py

```diff
@@ -109,39 +111,31 @@ def _query(
 
         return json_ret
 
-    def _get_raw_transactions(self, signatures: list[str]) -> list[dict[str, Any]] | None:
-        """Query Helius for the raw transactions corresponding to the given signatures.
-        Returns a list of raw transaction dicts or None if there was a problem querying the API.
-        """
-        raw_transactions = []
-        try:
-            for chunk in get_chunks(signatures, MAX_TX_BATCH_SIZE):
-                raw_transactions.extend(self._query(
-                    endpoint='transactions',
-                    params={'transactions': chunk},
-                ))
-        except (RemoteError, MissingAPIKey) as e:
-            log.error(f'Failed to query {len(signatures)} transactions from Helius due to {e!s}')
-            return None
-
-        return raw_transactions
-
-    def get_transactions(self, signatures: list[str]) -> list[SolanaTransaction] | None:
-        """Query Helius for solana transactions corresponding to the given signatures.
-        Returns a list of SolanaTransactions or None if there was a problem querying the API.
+    def get_transactions(self, signatures: list[str], relevant_address: 'SolanaAddress') -> None:
+        """Query Helius for txs corresponding to the given signatures and save them in the DB.
+        May raise:
+        - RemoteError if there was a problem with the remote query.
+        - MissingAPIKey if the user has no Helius api key.
         """
-        if (raw_transactions := self._get_raw_transactions(signatures)) is None:
-            return None
-
-        txs = []
-        for raw_tx in raw_transactions:
-            try:
-                txs.append(self._deserialize_raw_tx(raw_tx))
-            except DeserializationError as e:
-                log.error(e)
-                continue
-
-        return txs
+        for chunk in get_chunks(signatures, MAX_TX_BATCH_SIZE):
+            txs = []
+            for raw_tx in self._query(
+                endpoint='transactions',
+                params={'transactions': chunk},
+            ):
+                try:
+                    txs.append(self._deserialize_raw_tx(raw_tx))
+                except DeserializationError as e:
+                    log.error(e)
```

Comment:
provide more information about the error. otherwise it will be hard to find the source